### ISSUE TITLE
Wrong argument order in training_with_built_in_methods.py

### ISSUE THREAD

**Issue by gqqnbig** [keras-team/keras-io#802]
https://github.com/keras-team/keras-io/blob/5aa03e8dba2cf2c3409a47e194ae1a6eb557e7c0/guides/training_with_built_in_methods.py#L445-L479

LogisticEndpoint.call is defined to take `(targets, logits)`. However this method is called as `LogisticEndpoint(...)(logits, targets)`. The argument order is reversed.

I don't know which is the correct order. Please fix.

**Comment by gqqnbig** [comment]
On https://www.tensorflow.org/guide/keras/custom_layers_and_models, the arguments and parameters of LogisticEndpoint are targets then logits.

**Comment by SuryanarayanaY** [comment]
Hi @gqqnbig ,

The proposed PR has been merged and up to date with latest version.
https://github.com/keras-team/keras-io/blob/74c762982f26081292fd0661f1736b72210fcf33/guides/training_with_built_in_methods.py#L423

Could you please feel free to close the issue now. Thanks!

**Comment by gqqnbig** [comment]
LGTM, thank you!
